go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/gatling/frontline-cloud-doc
 
 go 1.18
 
-require github.com/gatling/gatling.io-doc-theme v0.0.0-20231107094112-f6b5d70cfe07 // indirect
+require github.com/gatling/gatling.io-doc-theme v0.0.0-20231120163357-2bf3be01f70c // indirect
```

Comment:
Might get some conflicts on that o//

Reply:
I have to go mod update again ?

Reply:
Yes, you also don't have to commit this change!